common/beacon.go

```diff
@@ -12,6 +12,9 @@ const DefaultChainHash = "default"
 // MultiBeaconFolder is the name of the folder where the multi-beacon data is stored
 const MultiBeaconFolder = "multibeacon"
 
+// RateLimit is used to reduce log verbosity when doing bulk processes, issuing logs only every RateLimit beacons
+const RateLimit = 300
```

Comment:
* this isn't the definition i would expect for ratelimit - which is closer to 'i want n logs per time'. having a larger rate limit here means less logs.
* This is a bad name for this constant - it is not a generic `RateLimit` for the common beacons. it's a logging / debug helper at best.

Reply:
Renamed `LogsToSkip` since a percentile version wasn't always possible, e.g. when in a process that doesn't know "when it ends".